Dockerfile

```diff
@@ -23,6 +23,9 @@ RUN docker-php-ext-install mcrypt \
       intl \
       pgsql
 
+RUN pecl install -f memcache
```

Comment:
Better to use `memcached`. Pecl will use the same source `https://github.com/php-memcached-dev/php-memcached`. I don't see any problem in working with Pecl.